### PR TITLE
Release note/Control coverage(210715)

### PR DESCRIPTION
**Pull request by iamPallav**

### CODE REVIEW

File: Release notes/LatestReleaseNotes.md [hunk 1]
```diff
@@ -1,72 +1,79 @@
...
+    * Added a switch ‘-PrepareForFix’ which will create a local backup of current state of a resource     configuration/settings.
+        ```PowerShell
+        gads - oz $org -pn $proj  -Feednames * -cid ‘ADO_Feed_AuthZ_Restrict_Broader_Group_Access’ -upc    -PrepareForControlFix
+        ```	
+    * Thereafter, a new command, Set-AzSKADOSecurityStatus, can be used to fix the control for which a ‘state backup’ has been generated via the previous command.
+        ```PowerShell
+        Set-AzSKADOSecurityStatus -oz $org -pn $proj -cid ’ADO_Feed_AuthZ_Restrict_Broader_Group_Access’
         ```
-    * To reset the org policy to default location: 
+    * Currently, this is available for the ADO_Feed_AuthZ_Restrict_Broader_Group_Access control.
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:28]: Add ``` ```` in the control mentioned

File: Release notes/LatestReleaseNotes.md [hunk 2]
```diff
@@ -1,72 +1,79 @@
...
+* Optimizing checkpointed scans: 
+    * Added a switch ‘-DoNotRefetchResources’ (or -dnrr) to request not re-fetching resources when continuation of a checkpointed scan (i.e., when the -upc switch is used).
+    * This facilitates running batch scans efficiently by speeding up scans of subsequent batches after the initial one.
+
+* Automated control fixes (preview): 
+    * Added a switch ‘-PrepareForFix’ which will create a local backup of current state of a resource     configuration/settings.
+        ```PowerShell
+        gads - oz $org -pn $proj  -Feednames * -cid ‘ADO_Feed_AuthZ_Restrict_Broader_Group_Access’ -upc    -PrepareForControlFix
+        ```	
+    * Thereafter, a new command, Set-AzSKADOSecurityStatus, can be used to fix the control for which a ‘state backup’ has been generated via the previous command.
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:24]: add ``` ``` in Set-AzSKADOsecuritystatus

File: Release notes/LatestReleaseNotes.md [hunk 3]
```diff
@@ -1,72 +1,79 @@
...
+        ```PowerShell
+        Set-AzSKADOSecurityStatus -oz $org -pn $proj -cid ’ADO_Feed_AuthZ_Restrict_Broader_Group_Access’
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:26]: **’**ADO_Feed_AuthZ_Restrict_Broader_Group_Access’

File: Release notes/LatestReleaseNotes.md [hunk 4]
```diff
@@ -1,72 +1,79 @@
...
 
 ## Feature Updates
 
 ### ADO Security Scanner PowerShell Module:
-* Local folder-based org policy support:
-    * To facilitate use of ADO Scanner locally (from desktop console) for driving compliance for an org, we have added support to run against org policy (custom control settings, etc.) from a local folder. This capability can be used by org/project admins to evaluate ADO Scanner and fine tune its configuration for their org/environment before deploying the org policy as a policy repo.
-    * Command to configure a local folder as source of org policy:
+* Support for ‘path’ based scans:
+    * To support scans of ‘compliant folder’ based projects (where governed build/release configs are       constrained to specific paths), we have added support for path-based scanning for builds and releases.       
+    * This can be used via the -BuildsFolderPath and -ReleasesFolderPath switches as shown below:
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:8]: Add ``` ```` for -BuildsFolderPath and -ReleasesFolderPath switches

File: Release notes/LatestReleaseNotes.md [hunk 5]
```diff
@@ -1,72 +1,79 @@
...
+        
         ```PowerShell
-        Set-AzSKADOPolicySettings -LocalOrgPolicyFolderPath $folderpath
+        gads –oz $org -pn $proj –bn *  –BuildsFolderPath $folderpath
+        
+        gads –oz $org –pn $proj –rn *  –ReleasesFolderPath $folderpath
+        ``` 
+* Optimizing checkpointed scans: 
+    * Added a switch ‘-DoNotRefetchResources’ (or -dnrr) to request not re-fetching resources when continuation of a checkpointed scan (i.e., when the -upc switch is used).
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:16]: same for -DoNotRefetchResources

**Review by v-himkam — COMMENTED** [review]
please address the comments

File: Release notes/LatestReleaseNotes.md [hunk 6]
```diff
@@ -1,72 +1,83 @@
...
+    * Added a switch ```-DoNotRefetchResources``` (or -dnrr) to request not re-fetching resources when continuation of a checkpointed scan (i.e., when the -upc switch is used).
+    * This facilitates running batch scans efficiently by speeding up scans of subsequent batches after the initial one.
+
+* Automated control fixes (preview): 
+    * Added a switch ```-PrepareForFix``` which will create a local backup of current state of a resource     configuration/settings.
+        ```PowerShell
+        gads - oz $org -pn $proj  -Feednames * -cid 'ADO_Feed_AuthZ_Restrict_Broader_Group_Access' -upc    -PrepareForControlFix
+        ```	
+    * Thereafter, a new command, 
+      ```PowerShell
```
Comment (v-himkam) [inline Release notes/LatestReleaseNotes.md:25]: command should be just inside `` ``` not in ```Powershell ``

**Review by v-himkam — COMMENTED** [review]
address one minor comment

**Review by v-himkam — APPROVED** [review]

**Review by v-soukum — APPROVED** [review]

**Pull request merged**